<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFilterWidget</class>
 <widget class="QWidget" name="TextFilterWidget">
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Enter command to pipe selected text through:</string>
     </property>
     <property name="buddy">
      <cstring>filterBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KHistoryComboBox" name="filterBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="copyResult">
     <property name="toolTip">
      <string>Copy the result to clipboard leaving a document unchanged.</string>
     </property>
     <property name="text">
      <string>Copy the result instead of pasting it</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mergeOutput">
     <property name="toolTip">
      <string>If checked, an output from STDOUT and STDERR will be merged and no errors will be reported.
Otherwise, STDERR will be displayed as a passive message.</string>
     </property>
     <property name="text">
      <string>Merge STDOUT and STDERR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
